integr8ly/integreatly-operator
# 3scale update to 0.8.3+0.1645735250.p

Update 3scale manifest to 0.8.3+0.1645735250.p 

This is an automatically generated branch to track the '0.8.3+0.1645735250.p' version of the '3scale' OLM manifests.
This branch will be automatically updated with any changes related to this version until it finally GAs. 

Changes required to fix installation issues should be applied to this branch following the usual [PR Workflow](https://github.com/integr8ly/delorean/blob/master/docs/integrealy-ci/prow-pr-workflow.md)

More details about the Early Warning System that generated this can be found [here](https://github.com/integr8ly/delorean/tree/master/docs/ews)


## To verify
- Install RHOAM via olm from this index image: ```quay.io/mstoklus/integreatly-index:1.18.2```
- Once installed, setup workload web app on your cluster
- Update catalog source to point to a new index - ```quay.io/mstoklus/integreatly-index:1.19.0```
- Approve the upgrade
- Wait for upgrade to complete and check the total downtime in 3scale (should not be longer than 2mins)
- Confirm upgrade completed successfully. 


## Type of change

- [X] Product Update

## Checklist
- [ ] Older product manifests (N-2) have been removed
- [ ] Operator and Product Versions are updated correctly
- [ ] Go Modules are updated if required
